bower.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "angular": "*",
-    "app-logger-angular": "1.1.0",
+    "app-logger-angular": "~2.0",
```

Comment:
~2.0.0 isn't what we want.  ~2.0.0 will allow 2.0.1, 2.0.2 etc but not 2.1.0.  Minor versions (e.g. 2.1.0) should be backward compatible so that's what I'm indicating here.